frontend/src/services/ragService.ts

```
@@ -0,0 +1,78 @@
+import { Collection } from '@/pages/DocumentManager/types';
+import apiClient from '../lib/utils/apiClient';
+
+export const getCollections = async () => {
+  try {
+    const response = await apiClient.get('/rag/collections');
+    return response.data;
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)